### PR TITLE
Mh webscraping

### PR DESCRIPTION
**Pull request by holmesm8**
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Notes

There are two separate files here, a webscraper for trending books for Goodreads and a moviescraper from IMDB's top 250 movies list. I've refactored and gotten them to both return a json object.

I think this could be a good first step in adding to our features (just getting it to work). We'll definitely want to look into setting up a database and possibly using the webscrapers as an automated task that runs once a week or once daily.

We also don't need to keep these in the file locations they are currently in. Once we get a better understanding of the structure we can rearrange as needed.

### CODE REVIEW

File: movie_webscraper.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    place = movie[:len(str(index))-(len(movie))]
+    # data = {"movie_title": movie_title, "place": place}
+    imdb.append(movie_title)
+
+# THIS WILL PRINT THE ENTIRE 250 LIST
+# for item in imdb:
+# print(item['place'], '-', item['movie_title'])
+# print(item['movie_title'])
+
+random_movies = random.sample(imdb, 10)
```
Comment (holmesm8) [inline movie_webscraper.py:34]: This will limit our return of movies to 10. Keeping it consistent with the number of books that are returned (also 10)

**Comment by holmesm8** [comment]
@david-holtkamp @kathleen-carroll 

Would love for you two to take a look over these two files. If you'd like to set some time aside I can do a quick walkthrough of what each line is doing.

File: movie_webscraper.py (hunk 1 above)
Comment (Will-Meighan) [reply movie_webscraper.py:34]: I refactored the events API call down to five, but I think this is okay to make it easier to work with.

**Comment by Will-Meighan** [comment]
This looks great! Next steps are to combine the webscrape with the API call. Talked it over in person and for now are planning on returning one json object with all events, books, and movies. Also need to figure out testing.

**Pull request merged**